app/api/schema/tickets.py

```diff
@@ -29,11 +29,22 @@ def validate_date(self, data, original_data):
 
             if 'sales_ends_at' not in data:
                 data['sales_ends_at'] = ticket.sales_ends_at
+         
+            if 'event_ends_at' not in data:
+                data['event_ends_at'] = ticket.event.ends_at
 
         if data['sales_starts_at'] >= data['sales_ends_at']:
             raise UnprocessableEntity({'pointer': '/data/attributes/sales-ends-at'},
                                       "sales-ends-at should be after sales-starts-at")
 
+        if data['sales_starts_at'] > data['event_ends_at']:
```

Comment:
Put these  in two conditions in
```python

if event_ends_at in data

```
maybe this is the reason of validation error?

Reply:
Thanks @uds5501. It helped :)